src/index.html

```diff
@@ -3,11 +3,302 @@
+            <p class="subtitle__date">26.08-29.11.2019</p>
+          </div>
+          <h3 class="card__title">
+            Йду i повертаюсь
+          </h3>
+          <p class="card__text">
+            Національний Художній Музей України презентує унікальну частину колекції Градобанку - українське мистецтво другої
+            половини 1980-1995 років.
+          </p>
+        </div>
+        <div class="events__card card">
+          <picture class="card__img">
+            <source media="(min-width: 1280px)" srcset="./images/1280/03.png">
+            <source media="(min-width: 768px)" srcset="./images/768/03.png">
+            <img src="./images/320/03.png" alt="picture">
+          </picture>
+          <div class="card__subtitle subtitle">
+            <p class="subtitle__event">ВИСТАВКА</p>
+            <p class="subtitle__date">26.08-29.11.2019</p>
+          </div>
+          <h3 class="card__title title">
+            I спогади i мрії
+          </h3>
+          <p class="card__text">
+            Національний художній музей України до 100 річчя від дня народження видатної української художниці Тетяни Яблонської
+            відкриває ретроспективну ювілейну виставку «І спогади i мрії».
+          </p>
+        </div>
+        <div class="events__advert">
+          <div class="events__img"></div>
+          <div class="events__box">
+            <div class="events__subtitle subtitle">
+              <p class="subtitle__event">лекція</p>
+              <p class="subtitle__date">26/08/2019 - 11:00</p>
+            </div>
+            <h3 class="events__name ">Від класицизму до романтизму</h3>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- gallery -->
+    <section class="gallery" id="gallery">
+      <div class="gallery__container">
+        <h3 class="gallery__title title">
+          Галерея
+        </h3>
+
+        <div class="gallery__box gallery__container">
+          <img src="./images/1280/05.png"
+          alt="picture" class="gallery__p1">
+          <img src="./images/768/05.1.png"
+            alt="p"
+            class="gallery__p">
+          <img src="./images/1280/05.1.png"
+          alt="picture" class="gallery__p2">
+          <img src="./images/1280/05.2.png"
+          alt="picture" class="gallery__p3">
+          <img src="./images/1280/05.3.png"
+          alt="picture" class="gallery__p4">
+        </div>
+      </div>
+    </section>
+    <!-- subscribe -->
+    <section class="subscribe">
+      <div class="subscribe__container">
+        <h3 class="subscribe__title title">
+          Підписка
+        </h3>
+        <p class="subscribe__text">
+          Бути в курсі всіх актуальних подій та останніх новин.
+        </p>
+        <form action=" "
+          class="subscribe__form"
+          method="get"
+        >
+          <input type="email"
+            class="subscribe__input"
+            placeholder="E-mail"
+            required
+          >
+          </input>
```

Comment:
Stray end tag input. This doesn't match the validation rules. For checking HTML validity use https://validator.w3.org/